### PR TITLE
Remove the pseudostate focus outline on buttons - closes #28

### LINKED ISSUE #28: Remove Square Border on Focus for Emotion Try Buttons

### PR DESCRIPTION
**Pull request by letakeane**
## What's this PR do?

_Remove the outline from the :focus pseudoselector._

## Pre-Merge TODOs
 - [x] Screenshots Included

## Learning

_Had to google to double check how to target the focus pseudoselector._
_Opened up the dev tools to see how :focus was styled (spoiler alert: it just has a prop of "outline"._

## Where should the reviewer start?

_Not much to review!_

## Screenshots (if appropriate)
Before:
before, with focus outline

After:
after, with transparent focus outline

Focus!
FOCUS!!
FOCUS!!!
FOCUS!!!!

### CODE REVIEW

**Comment by anderswood** [comment]
Excellent documentation, I approve.

File: app/lib/analyzeResponse.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const analyzeResponse = (data) => {
+const analyzeResponse = (data) => {
   let scores = data[0].scores;
   let emotions = Object.keys(scores);
   let sortedEmotions = emotions.sort((a, b) => {
```
Comment (Jeff-Duke) [inline app/lib/analyzeResponse.js]: these can/should all be `const` instead of `let` since you're not reassigning them.

Comment (letakeane) [inline app/lib/analyzeResponse.js]: Perfect - thank you @Jeff-Duke !

**Comment by lauraturk** [comment]
Sweet! Now I've learned how to do that too!

**Pull request merged**